roles/build_openstack_packages/tasks/create_repo.yml

```diff
@@ -14,24 +14,31 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Create a directory to hold the rpms
-  ansible.builtin.file:
-    path: '{{ cifmw_bop_gating_repo }}'
+- name: Create a temproary directory to hold the rpms
```

Comment:
nit s/temproary/temporary

Reply:
same change exists in https://github.com/openstack-k8s-operators/ci-framework/pull/1457 are you going to merge them into one review or update one of them/rebase?